patroni/ha.py

```diff
@@ -491,7 +526,10 @@ def reinitialize(self, cluster):
 
     def process_scheduled_action(self):
         if self.reinitialize_scheduled():
-            if self.cluster.is_unlocked():
+            if self.is_paused():
+                logger.warning('Cluster is in a pause state, can not reinitialize')
```

Comment:
I actually see a use-case for bootstrap when Patroni is disabled, namely, when the master node is upgraded and then restarted to initialize all replicas one by one.